vividus-plugin-web-app-to-rest-api/src/main/resources/vividus-plugin/spring.xml

```diff
@@ -7,10 +7,32 @@
         http://www.springframework.org/schema/util https://www.springframework.org/schema/util/spring-util.xsd"
         default-lazy-init="true" profile="web">
 
+    <bean id="httpClient-WebAppToRestApi" class="org.vividus.http.client.HttpClient"
+          factory-bean="httpClientFactory"
+          factory-method="buildHttpClient">
+        <constructor-arg>
+            <bean parent="httpClientConfig">
+                <property name="headers">
+                    <bean factory-bean="propertyParser" factory-method="getPropertyValuesByPrefix">
+                        <constructor-arg value="rest-api.http.header." />
+                    </bean>
+                </property>
+                <property name="httpResponseHandlers"><list/></property>
+            </bean>
+        </constructor-arg>
+    </bean>
+
     <bean id="httpRequestSteps" class="org.vividus.steps.integration.HttpRequestSteps" />
 
     <bean id="resourceCheckSteps" class="org.vividus.steps.integration.ResourceCheckSteps" init-method="init">
         <property name="uriToIgnoreRegex" value="${resource-checker.uri-to-ignore-regex}" />
+        <constructor-arg name="httpRequestExecutor">
```

Comment:
hmm, it seems this executor is used to download pages to parse, but we should attach the downloaded pages and skip attaching of resource HTTP messages only